core/lib/multivm/src/versions/vm_latest/implementation/execution.rs

```diff
@@ -49,6 +47,7 @@ impl<S: WriteStorage, H: HistoryMode> Vm<S, H> {
             tracers,
             self.storage.clone(),
             refund_tracers,
+            Some(PubdataTracer::new(self.batch_env.clone(), execution_mode)),
```

Comment:
why do we need some if it's always enabled?

Reply:
what is the custom implementation?

Reply:
For instance, the bootloader tester written by @mm-zk: https://github.com/matter-labs/era-system-contracts/tree/main/bootloader/test_infra

Reply:
I don't think that @mm-zk 's tooling would be broken if the pubdata tracer is always put there, though still I thought that our general approach is to preserve choices for the users of the VM

Reply:
ok... just less checks is a bit better performance wise :)

Reply:
I don't have strong opinions here (as long as this tracer doesn't prevent test bootloader from running)